lib/DynamoResponse.js

```diff
@@ -57,8 +57,9 @@ var DynamoResponse = function (tablePrefix, output, repeatWithStartKey) {
     this.result = typeUtil.unpackObjectOrArray(output.Item)
 
   // for DeleteItem, PutItem and UpdateItem, 'result' is {Object}
-  } else if (output.Attributes) {
-    this.result = typeUtil.unpackObjectOrArray(output.Attributes)
+  } else if (output.UpdatedAttributes) {
+    this.result = typeUtil.unpackObjectOrArray(output.UpdatedAttributes)
```

Comment:
it would be nicer to just replace the updated attributed base off `this.previous`?